CSS/style.css

```diff
@@ -0,0 +1,736 @@
+@import url(https://fonts.googleapis.com/css?family=Poppins:100,100italic,200,200italic,300,300italic,regular,italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic);
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+}
+body {
+    background-color: #151515;
+}
```

Comment:
- [ OPTIONAL ] Consider using CSS variables (custom properties) to define colors, font families, and other common values. This can make it easier to update the styling of your website globally.

sample:

```
/* Define CSS variables for colors */
:root {
    --primary-color: #21243D;
    --button-background: #FF6464;
    --button-border: #FF6464;
    --button-text: #FFF;
}

/* Rest of your CSS styles */
.title {
    color: var(--primary-color);
    /* ...other styles... */
}

.hero-btn {
    border-radius: 2px;
    background: var(--button-background);
    border: 1px solid var(--button-border);
    color: var(--button-text);
    /* ...other styles... */
}
```

**you can read more about that [here](https://www.freecodecamp.org/news/what-are-css-variables-and-how-to-use-them/)**